indra/assemblers/html/assembler.py

```diff
@@ -637,7 +638,8 @@ def get_role(ag_ix):
                         'original_json': ev.to_json(),
                         'num_curations': num_curations,
                         'num_correct': num_correct,
-                        'num_incorrect': num_incorrect
+                        'num_incorrect': num_incorrect,
+                        'source_url': ev.annotations.get('source_url', '')
```

Comment:
why not just splat in all `ev.annotations`?

Reply:
oh... there could potentially be all sorts of stuff there

Reply:
I guess we could, although it is already accessible under `'original_json': ev.to_json()`